package.json

```diff
@@ -10,24 +10,20 @@
   ],
   "funding": "https://opencollective.com/unts/projects/eslint-import-resolver-ts",
   "license": "ISC",
-  "packageManager": "yarn@4.0.2",
+  "packageManager": "yarn@4.7.0",
   "engines": {
-    "node": "^14.18.0 || >=16.0.0"
+    "node": "^16.17.0 || >=18.6.0"
```

Comment:
So a breaking change then.

Reply:
Yes, next major version.

Reply:
It seems we can not drop `@nolyfill/is-core-module` #305, then node 14 can still be supported.

Reply:
Let's do `isBuiltin`!

Reply:
I'm not sure I understand the discussion - is it about keeping support for node 14? [isBuiltin](https://nodejs.org/api/module.html#moduleisbuiltinmodulename) is only available from node 16.17.

Reply:
> I'm not sure I understand the discussion - is it about keeping support for node 14? [isBuiltin](https://nodejs.org/api/module.html#moduleisbuiltinmodulename) is only available from node 16.17.

The next major drops Node 14 support. Are you still using Node 14, with `eslint-plugin-import`?

Reply:
No, sorry, I was just confused about the goal :) Thankfully node 14 support is not something my team requires anymore.